### PR TITLE
Improve the baseline/ regression diff for images of different height

### PR DESCRIPTION
**Pull request by solker**
#61

### CODE REVIEW

**Comment by amitaibu** [comment]
Screenshot please :)

File: client/app/scripts/controllers/screenshots.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ angular.module('clientApp')
 
     angular.forEach($scope.screenshots, function(value, key) {
       $scope.screenshots[key].selected = false;
+      // Find the highest picture.
```
Comment (amitaibu) [inline client/app/scripts/controllers/screenshots.js]: picture => image

**Comment by solker** [comment]
@amitaibu 
merge please.
screenshot from 2015-06-16 16 59 33
screenshot from 2015-06-16 16 59 20

File: client/app/styles/_before_after_image_slider.scss [hunk 2]
```diff
@@ -1,11 +1,31 @@
+.labels {
+  height: 25px;
+  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
```
Comment (solker) [inline client/app/styles/_before_after_image_slider.scss]: Why `@font-family-monospace` [link](http://getbootstrap.com/css/#less-variables-typography) doesn't work here?

**Comment by solker** [comment]
@amitaibu ready for code review.

shoov_before_after

File: client/app/scripts/directives/beforeafterimageslider.js [hunk 3]
```diff
@@ -10,30 +10,37 @@ angular.module('clientApp')
...
+          init_split = Math.round(scope.maxWidth/2);
 
-        element.find('.before-after-slider').mousemove(function(e){
-          var offX  = (e.offsetX || e.clientX - $second_wrapper.offset().left);
-          $second_wrapper.width(offX);
+        $baseline_wrapper.width(init_split);
+        
+        element.find('.before-after-slider').mousemove(function(e) {
+          var offX  = (e.offsetX || e.clientX - $baseline_wrapper.offset().left);
+          console.log(offX);
```
Comment (amitaibu) [inline client/app/scripts/directives/beforeafterimageslider.js]: remove

File: client/app/styles/_before_after_image_slider.scss [hunk 4]
```diff
@@ -1,11 +1,31 @@
+.labels {
```
Comment (amitaibu) [inline client/app/styles/_before_after_image_slider.scss]: move inside the `.before-after-slider`?

File: client/app/scripts/directives/beforeafterimageslider.js [hunk 5]
```diff
@@ -10,30 +10,37 @@ angular.module('clientApp')
...
       },
       template:
-        '<div class="before-after-slider">' +
-            '<div class="first-wrapper">' +
-              '<img ng-src="{{ first }}" width="{{ width }}" height="{{ height }}" alt="first" />' +
-            '</div>' +
-          '<div class="second-wrapper">' +
-            '<img ng-src="{{ second }}" width="{{ width }}" height="{{ height }}"  alt="second" />' +
+        '<div class="labels" style="width: {{ maxWidth }}px">' +
+          '<div class="baseline text-muted">baseline image</div>' +
```
Comment (amitaibu) [inline client/app/scripts/directives/beforeafterimageslider.js]: baseline image => Baseline

File: client/app/scripts/directives/beforeafterimageslider.js [hunk 6]
```diff
@@ -10,30 +10,37 @@ angular.module('clientApp')
...
+          '<div class="regression text-muted">regression image</div>' +
```
Comment (amitaibu) [inline client/app/scripts/directives/beforeafterimageslider.js]: regression image => Regression

**Comment by amitaibu** [comment]
Is the `default.arduino.baseline.png` file necessary?

**Comment by solker** [comment]
@amitaibu excuse me, check again please.

**Comment by amitaibu** [comment]
When switching to diff mode, there should be still a black border

localhost_9000__code_966a34cb1d4040a7c2af__screenshots_22-2

File: client/app/styles/_before_after_image_slider.scss [hunk 7]
```diff
@@ -4,8 +4,38 @@
...
+    }
+  }
+
+  .baseline-wrapper {
+    height: 100%;
+    border-left: 2px solid green;
+    border-top: 2px solid green;
+    border-bottom: 2px solid green;
+    border-right: 1px solid black;
+    background-color: transparent;
```
Comment (solker) [inline client/app/styles/_before_after_image_slider.scss:30]: It's fix for half display. If you do not want transparent we can change it to any another color or picture.

**Comment by solker** [comment]
@amitaibu 
I fixed border also solved the problem with a half of screen.
Check this line of code, we can set background color, set picture or use any color to determine that image it's not full. https://github.com/shoov/shoov/pull/62/files#r32624599

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request closed without merging**